src/app/ReadClient.cpp

```diff
@@ -663,6 +682,32 @@ void ReadClient::OnResponseTimeout(Messaging::ExchangeContext * apExchangeContex
     Close(CHIP_ERROR_TIMEOUT);
 }
 
+CHIP_ERROR ReadClient::ReadICDOperationModeFromAttributeDataIB(const TLV::TLVReader & aReader, PeerType & aType)
+{
+    TLV::TLVReader reader;
+    reader.Init(aReader);
+
+    Clusters::IcdManagement::Attributes::OperatingMode::TypeInfo::DecodableType operatingMode;
```

Comment:
So this calls the attribute OperatingMode.  Elsewhere in documentation and the name of this function we use OperationMode.  Which is it?  Should be consistent.

Reply:
It should be OperatingMode, thanks!